docs/software/remote_desktop.md

```diff
@@ -33,80 +34,212 @@
+
+=== "VNC Server settings"
+
+    #### VNC Server - Shared desktop
+
+    The *shared desktop* mode is used to have more than one single VNC viewer connected to the VNC server. To enable this mode, edit dietpi.txt (e.g. via `nano /boot/dietpi.txt`).  
+    Change to value 1 the following line
 
-    - Use the IP address of your DietPi device (e.g.: `192.168.0.100`). If you can't connect, try connecting to screen 1 (e.g.: `192.168.0.100:1`)
-    - Use the password you entered during the installation. If you would like to change the password run from the console/terminal
     ```
-    vncpasswd
+    SOFTWARE_VNCSERVER_SHARE_DESKTOP=1
     ```
-    - The default port is 5901. **Note:** To enable access from outside of your local network, this port needs to be forwarded from your router.
 
-=== "Enable autostart"
+    A running desktop is required in this mode, therefore set the desktop to autostart from boot via `dietpi-autostart`.
 
-    VNC servers start automatically during boot, unless you have selected **Desktop** from `dietpi-autostart` configuration tool.
+    #### VNC Server - Resolution settings
+
+    The following shows an example how to run VNC server on screen :1 by creating a new desktop with 1280x720 resolution by editing dietpi.txt:
 
-    To enable a different startup setup, run from the command line the main DietPi tool:
     ```
-    dietpi-launcher
+    nano /boot/dietpi.txt
     ```
-    or directly.
+
+    and change the following settings:
+
     ```
-    dietpi-autostart
+    SOFTWARE_VNCSERVER_WIDTH=1280
+    SOFTWARE_VNCSERVER_HEIGHT=720
+    SOFTWARE_VNCSERVER_DEPTH=32
+    SOFTWARE_VNCSERVER_DISPLAY_INDEX=1
     ```
-    To see all the DietPi configurations options, review [DietPi Tools](../../dietpi_tools) section.
+
+    At last restart the service to activate the new settings:
+
+    ```
+    systemctl restart vncserver.service
+    ```
+
+=== "Enable autostart"
+
+    VNC servers start automatically during boot, unless you have selected **Desktop** from the `dietpi-autostart` configuration tool.
+
+    To enable a different startup setup, run from the command line the main DietPi tool `dietpi-autostart`. To see all the DietPi configurations options, review the [DietPi Tools](../../dietpi_tools) section.
 
 === "Official documentation"
-    [LXDE Website](https://www.lxde.org) - [LXDE Wiki](https://wiki.lxde.org/en/Main_Page)
 
-![DietPi-Remote-Desktop-TigerVNC](../assets/images/dietpi-software-remote-desktop-tigervnc.jpg)
+    - [LXDE Website](https://www.lxde.org)
+    - [LXDE Wiki](https://wiki.lxde.org/en/Main_Page)
+
+## RealVNC Server - Desktop for remote connection
+
+RealVNC consists of the *VNC Server* and the *VNC Viewer* application to share the desktop or control the computer running the VNC Server.
+
+![DietPi remote desktop software RealVNC](../assets/images/dietpi-software-remotedesktop-realvnc.png){: style="width:600px"}
+
+=== "VNC Server modes"
+
+    #### Basics
```

Comment:
Empty line required here to satisfy markdownlint.

Reply:
Ok, changed.